content/_layouts/docs.njk

```diff
@@ -10,19 +10,31 @@ layout: default
   <div id="responsive-menu"><button class='btn btn-primary' onclick='handleMenu()'>Table of Contents</button></div>
   <div id="content" class="row-fluid">
     <div class="toc well span3 visible-toc">
+      <div style='display: flex; flex-direction: row; align-items: space-between; justify-content: space-between;'>
```

Comment:
nit pick: would be good to use double quotes everywhere, looks like this file has mixed styles